package.json

```diff
@@ -290,23 +290,31 @@
               "type": "java",
               "name": "Debug (Launch)",
               "request": "launch",
-              "cwd": "^\"\\${workspaceFolder}\"",
               "console": "internalConsole",
-              "stopOnEntry": false,
               "mainClass": "",
               "args": ""
             }
           },
+          {
+            "label": "Java: Launch Program in Current File",
+            "description": "Add a new configuration for launching current java file.",
+            "body": {
+              "type": "java",
+              "name": "Debug (Launch) - Current File",
+              "request": "launch",
+              "console": "internalConsole",
```

Comment:
I think we can hide the console and args, see the sample generated by node.js
```
{
            "type": "node",
            "request": "launch",
            "name": "Launch Program",
            "program": "${workspaceFolder}/app.js"
        }
```

Reply:
I agree with Andy here. Make our configuration consist with other language and common VSCode settings. The console/args should be familiar to the customer if they are using VSCode.

Reply:
Removed `console`, `args`, and `attach` config from the default launch.json, and use the Config Snippet to improve the discovery.